File: GameStore.Domain/Entities/Game.cs
using GameStore.Domain.Common;

namespace GameStore.Domain.Entities;

public class Game : BaseEntity
{
    public string Key { get; set; }

    public string Name { get; set; }

    public string Description { get; set; }

    public decimal Price { get; set; }

    public List<Comment> Comments { get; set; } = new List<Comment>();

    public List<Genre> Genres { get; set; } = new List<Genre>();

    public List<PlatformType> PlatformTypes { get; set; } = new List<PlatformType>();
}

File: GameStore.Client/src/app/models/game.ts
import {Image} from "./image";
import {Genre} from "./genre";
import {PlatformType} from "./platformType";

export interface Game {
    id: number;
    key: string;
    name: string;
    description: string;
    price: number;
    image: Image;
    genres: Genre[];
    platformTypes: PlatformType[];
}
